node/cli/src/command.rs

```diff
@@ -65,7 +65,7 @@ impl<T: sc_service::ChainSpec + 'static> IdentifyChain for T {
 fn load_spec(id: &str) -> std::result::Result<Box<dyn ChainSpec>, String> {
 	match id {
 		#[cfg(feature = "frequency")]
-		"frequency-bench" => return Ok(Box::new(chain_spec::frequency::frequency())),
+		"frequency-bench" => return Ok(Box::new(chain_spec::frequency::benchmark_mainnet_config())),
```

Comment:
This is just a rename to remove the confusion as to why this function was still around.